middle_end/flambda2/simplify_shared/slot_offsets.ml

```diff
@@ -369,12 +370,13 @@ module Greedy = struct
 
   let print_set fmt s =
     Format.fprintf fmt
-      "@[<v 2>%d:@ @[<v>first_slot_after_closures: %d;@ \
-       first_slot_used_by_envvar: %d;@ unallocated_closures: @[<v>%a@];@ \
-       unallocated_env_vars: @[<v>%a@];@ allocated: @[<v>%a@]@]@]"
-      s.id s.first_slot_after_closures s.first_slot_used_by_envvar
-      print_slot_descs s.unallocated_closure_slots print_slot_descs
-      s.unallocated_env_var_slots print_slots s.allocated_slots
+      "@[<v 2>%d:@ @[<v>first_slot_after_function_slots: %d;@ \
+       first_slot_used_by_value_slots: %d;@ unallocated_function_slots: \
+       @[<v>%a@];@ unallocated_value_slots: @[<v>%a@];@ allocated: \
+       @[<v>%a@]@]@]"
+      s.id s.first_slot_after_function_slots s.first_slot_used_by_value_slots
+      print_slot_descs s.unallocated_function_slots print_slot_descs
+      s.unallocated_value_slots print_slots s.allocated_slots
```

Comment:
This function should be properly formated, and annotated with the attribute prevent ocamlformat from messing with the formatting.

Reply:
I'm not sure it's worth it in this case, the format string is fairly simple.